.gitignore

```diff
@@ -9,6 +9,8 @@ xcuserdata
 /platform/node/test/actual
 /platform/node/test/diff
 /mason_packages
+/benchmark/fixtures/api/cache.db
+/benchmark/fixtures/api/cache.db-*
```

Comment:
Yeah, I think making a copy of the DB works fine. I've noticed this error too; thanks for working on it!